app/ui/browser-modern/views/browser/page/index.jsx

```diff
@@ -111,10 +111,8 @@ class Page extends Component {
     });
 
     this.webview.addEventListener('page-title-set', e => {
+      // Not emitted when pages don't have a title.
       this.props.dispatch(PageActions.setPageDetails(this.props.pageId, { title: e.title }));
-      // Make sure we record this page navigation in the remote history
-      // only after we have a title available.
-      this.props.dispatch(ProfileEffects.addRemoteHistory(this.props.pageId));
     });
 
     this.webview.addEventListener('page-favicon-updated', e => {
```

Comment:
What's the rational behind waiting for a favicon to set it to history? To ignore fast loading interstitials? Whatever the reason is, should explain why here. Also what if the favicon changes, like a new badge added because you got an email, is it duplicate entry? Pages that don't have a favicon?